app/controllers/stellar_multisig/rails/api/v1/totp_controller.rb

```diff
@@ -0,0 +1,6 @@
+require_dependency "stellar_multisig/rails/application_controller"
+
+module StellarMultisig::Rails
```

Comment:
@jaspervanbrian it looks having `::Rails` for a Rails engine [is recommended](https://stackoverflow.com/a/8961409/61018), but it bothers me that the migrations have a `rails` in the table name.

Reply:
I think what will work is to [set the engine name](https://edgeapi.rubyonrails.org/classes/Rails/Engine.html) like:

```ruby
module MyEngine
  class Engine < Rails::Engine
    engine_name "my_engine"
  end
end
```

Then, when you create migrations, `rails` should not be there anymore.